libs/full/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -753,40 +754,34 @@ namespace hpx { namespace util {
 
     std::size_t runtime_configuration::get_ipc_data_buffer_cache_size() const
     {
-        if (has_section("hpx.parcel"))
+        util::section const* sec = get_section("hpx.parcel.ipc");
+        if (nullptr != sec)
```

Comment:
Purely subjective, and I don't mind to keep it like this, but what about:
```suggestion
        if (util::section const* sec = get_section("hpx.parcel.ipc")
```
If you prefer this I don't mind doing the change.

Reply:
I can do that.